.github/workflows/ci.yml

```diff
@@ -22,8 +22,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Why removing Windows?

Reply:
Installation process with python3.10 and windows is a little tricky and in CI causes errors while trying to install matplotlib. Plus we don't use it in our e2e test suite triggered from the client